src/library.js

```diff
@@ -2700,6 +2713,14 @@ LibraryManager.library = {
   emscripten_longjmp: function(env, value) {
     _longjmp(env, value);
   },
+#if WASM_BACKEND == 1
+  // We need this wrapper from llvm wasm backend, but this is generated only
+  // in asm.js compilation, so.
```

Comment:
Where does `emscripten_longjmp_wrapper` come from in asm.js? I don't see any references to it in the emscripten or fastcomp codebases? So does this call to `_longjmp` call back into the asm/wasm module?
wrt the comment I think @azakai just meant that the sentence seems to end abruptly. And I guess it doesn't really say what it does, but maybe the full explanation doesn't belong in library.js anyway?